src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java

```diff
@@ -90,18 +90,18 @@ public class ContactInfoResponse extends InfoResponse {
     private IntPostalInfo intPostalInfo;
     private LocalPostalInfo locPostalInfo;
     private String voice;
-    private int voiceX;
+    private long voiceX;
     private String fax;
-    private int faxX;
+    private long faxX;
```

Comment:
That `x` in the XSD is a “token”, according to the schema definition. It’d be better to just use String rather than long on these two fields. Could you please change them as well as their getters to `String` type?